tests/phpt/migration_php8/009_blank_no_warning.php

```diff
@@ -0,0 +1,20 @@
+@kphp_runtime_should_not_warn
```

Comment:
I think, it has both tags?  
```
@ok kphp_runtime_should_not_warn
```
(need to test whether such an annotation would fit your expectations)

Reply:
It seems `@ok` will be superfluous here, because if there are any compilation errors, then `@kphp_runtime_should_not_warn` will catch them and the test will fail.

Reply:
It's confusing that an ok test is not marked with @ok, imho :)

Reply:
Added